test/util/backend_utils.hpp

```diff
@@ -20,187 +20,210 @@
 
 class Handle;
 
-namespace ov_runtime
+namespace ngraph
 {
-    class Backend;
-    class Tensor
+    namespace ov_runtime
     {
-    public:
-        std::vector<float> data;
-        PartialShape shape;
-        ngraph::element::Type type;
-
-        Shape get_shape() { return shape.to_shape(); }
-        PartialShape get_partial_shape() { return shape; }
-        explicit Tensor(ngraph::element::Type type, PartialShape ps)
-            : type(type)
-            , shape(ps)
+        class Backend;
+        class Tensor : public ngraph::runtime::Tensor
         {
-        }
-        explicit Tensor(ngraph::element::Type type, Shape ps)
-            : type(type)
-            , shape(ps)
+        public:
+            std::vector<int8_t> data;
+
+            explicit Tensor(ngraph::element::Type type, PartialShape ps)
+                : runtime::Tensor(std::make_shared<ngraph::descriptor::Tensor>(type, ps, ""))
+            {
+            }
+            explicit Tensor(ngraph::element::Type type, Shape ps)
+                : runtime::Tensor(std::make_shared<ngraph::descriptor::Tensor>(type, ps, ""))
+            {
+            }
+
+            /// \brief Write bytes directly into the tensor
+            /// \param p Pointer to source of data
+            /// \param n Number of bytes to write, must be integral number of elements.
+            void write(const void* p, size_t n) override
+            {
+                const int8_t* v = (const int8_t*)p;
+                if (v == nullptr)
+                    return;
+                data.resize(n);
+                for (size_t i = 0; i < n; ++i)
+                {
+                    data[i] = v[i];
+                }
+            }
+            /// \brief Read bytes directly from the tensor
+            /// \param p Pointer to destination for data
+            /// \param n Number of bytes to read, must be integral number of elements.
+            void read(void* p, size_t n) const override
+            {
+                int8_t* v = (int8_t*)p;
+                if (v == nullptr)
+                    return;
+                for (size_t i = 0; i < n; ++i)
+                {
+                    v[i] = data[i];
+                }
+            }
+        };
+    }
+
+    class Executable
+    {
+    private:
+        InferenceEngine::CNNNetwork network;
+        std::string device;
+
+    public:
+        Executable(std::shared_ptr<Function> func, std::string _device)
         {
+            auto opset = ngraph::get_opset1();
+            bool all_opset1(true);
+            for (auto& node : func->get_ops())
+            {
+                if (!opset.contains_op_type(node.get()))
+                    all_opset1 = false;
+            }
+
+            if (!all_opset1)
+            {
+                std::cout << "UNSUPPORTED OPS DETECTED!" << std::endl;
+                THROW_IE_EXCEPTION << "Exit from test";
+            }
+            else
```

Comment:
Remove `else`. Code flow would not continue past `THROW_IE_EXCEPTION` so no need for `else`.